picard/script.py

```diff
@@ -307,6 +308,22 @@ def register_script_function(function, name=None, eval_args=True,
     )
 
 
+def _compute(operation, operands):
+
+    operand_list = [operands.get('x'), operands.get('y')] + list(operands.get('args'))
+    int_operators = {'add': operator.add,
+                     'sub': operator.sub,
+                     'mul': operator.mul,
+                     'div': operator.div,
+                     'mod': operator.mod}
+    logic_operators = {'and': (lambda x, y: x and y),
+                       'or': (lambda x, y: x or y)}
+    if operation in int_operators:
+        return str(reduce(int_operators[operation], map(int, operand_list)))
+    elif operation in logic_operators:
```

Comment:
Instead of having a `if` here, i would just make 2 functions `_compute_int` and `_compute_logic` and call them appropriately.